### PR TITLE
add AG News dataset

### PR DESCRIPTION
**Pull request by jxmorris12**
adds support for the AG-News topic classification dataset

### CODE REVIEW

File: datasets/ag_news/ag_news.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  booktitle={NIPS},
+  year={2015}
+}
+"""
+
+_TRAIN_DOWNLOAD_URL = "https://raw.githubusercontent.com/mhjabreel/CharCnn_Keras/master/data/ag_news_csv/train.csv"
+_TEST_DOWNLOAD_URL = "https://raw.githubusercontent.com/mhjabreel/CharCnn_Keras/master/data/ag_news_csv/test.csv"
+
+
+class AGNewsConfig(nlp.BuilderConfig):
```
Comment (mariamabarham) [inline datasets/ag_news/ag_news.py:56]: I think this class can be removed see that it's not used? Or you can refer to the [squad](https://github.com/huggingface/nlp/blob/master/datasets/squad/squad.py) dataset on how you can use the `config class`

File: datasets/ag_news/ag_news.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        return nlp.DatasetInfo(
+            description=_DESCRIPTION,
+            features=nlp.Features(
+                {"text": nlp.Value("string"), "label": nlp.features.ClassLabel(names=["World", "Sports", "Business", "Sci/Tech"])}
+            ),
+            homepage="http://groups.di.unipi.it/~gulli/AG_corpus_of_news_articles.html",
+            citation=_CITATION,
+        )
+
+    def _vocab_text_gen(self, train_file):
```
Comment (mariamabarham) [inline datasets/ag_news/ag_news.py:83]: This function is not used also I think?

**Comment by mariamabarham** [comment]
Thanks @jxmorris12 for adding this adding. 
Can you please add a small description of the PR?

File: datasets/ag_news/ag_news.py (hunk 1 above)
Comment (jxmorris12) [reply datasets/ag_news/ag_news.py]: thanks, I removed it!

File: datasets/ag_news/ag_news.py (hunk 2 above)
Comment (jxmorris12) [reply datasets/ag_news/ag_news.py]: I removed this too! I thought I needed it, sorry

File: datasets/ag_news/ag_news.py (hunk 1 above)
Comment (mariamabarham) [reply datasets/ag_news/ag_news.py]: great, thanks

File: datasets/rotten_tomatoes/rotten_tomatoes.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
```
Comment (ghomasHudson) [inline datasets/rotten_tomatoes/rotten_tomatoes.py:14]: Did you mean to include rotten_tomatoes in here as well as AG News?

**Pull request merged**